pkgs/os-specific/linux/minimal-bootstrap/default.nix

```diff
@@ -163,7 +163,7 @@ lib.makeScope
 
     ln-boot = callPackage ./ln-boot { };
 
-    mes = lib.recurseIntoAttrs (callPackage ./mes { });
```

Comment:
Why was that even there? Should we remove it for the other packages too?

Reply:
@emilytrau?

Reply:
maybe tinycc-bootstrap could keep it if we exposed the previous stages
tinycc-musl returns an attrset with `compiler` and `libs` so probably useful there
tinycc-mes doesn't seem to need it

I think that's the remaining `recurseIntoAttrs` usages

Reply:
It was added so that ofborg would pick up `.compiler` and `.libs`, which is suppose is unnecessary as they will be built as dependencies elsewhere.